web-rdfs/source/http_handlers.cc

```diff
@@ -77,25 +99,24 @@ void get_handler(std::shared_ptr<HttpsServer::Response> response,
     delete_file_handler(response, path);
   } else if (!typeOfRequest.compare("OPEN")) {
     read_file_handler(response, path);
-  } else {
-    create_file_handler(response, request);
+  } else if (!typeOfRequest.compare("CREATE")) {
+    create_file_handler(response, path);
+  } else if (!typeOfRequest.compare("LISTSTATUS")) {
+    ls_handler(response, request, path);
   }
 }
```

Comment:
Since this is `else if`, we should probably have a 404 handler to ensure that every logic path at least causes the server to respond to the client. Probably not necessary to add it in this PR, but I'm tracking in #195.